frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -15,8 +15,8 @@
     },
     "properties": {
       "id": "knative-event-source",
-      "groupId": "developer-catalog",
-      "href": "/catalog?catalogType=EventSource",
+      "groupId": "eventing",
+      "href": "/catalog/ns/:namespace?catalogType=EventSource&provider=[\"Red+Hat\"]",
```

Comment:
you don't need to set default provider as this was the ask for 4.9 with community sources coming in with different provider